### DISCUSSION TITLE
Error in Installing Moose

### DISCUSSION THREAD

**Discussion by AmanKumar0301** [idaholab/moose#27059] [Q&A General]
Following errors were encountered while running the test during installation of moose. Please help.

![1](https://github.com/idaholab/moose/assets/163105499/6a74a638-6929-46d1-b9ae-b91f2a21b5c7)

**Comment by milljm** [comment]
Can you peruse here: https://mooseframework.inl.gov/help/troubleshooting.html
And follow the section on "Failing Tests", and report back?
